README.md

```diff
@@ -20,4 +20,39 @@
 </div>
 
 ## gofuzz
-**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
\ No newline at end of file
+**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
+
+## How to use
+1. (Once) Install the **gofuzz** CLI
+    ```shell
+    go install github.com/CodeIntelligenceTesting/gofuzz/cmd/gofuzz@latest
+    ```
+2. (Once) Add the **sanitizers** package as a dependency for the code base you want to test. 
```

Comment:
I feel like this could need some minor clarification. The command below should be executed from within the to be fuzzed go projects root directory, so the dependency is added to the `go.mod` file right?

Reply:
Added text for more clarification.